### ISSUE TITLE
Modular equation-based model of Air Handling Units

### ISSUE THREAD

**Issue by KremerMartin** [RWTH-EBC/AixLib#834]
**What is the problem?**

Currently we are using the AHU model provided by @PMehrfeld to calculate the energy consumption of the air handling unit in building simulation. This model is fixed in its configuration. We want to resolve this restriction, by providing an AHU model consisting of sub-models for the AHU components.

**How do we want to solve it? Describe the solution you'd like**

We will provide one model each for every component in the air handling unit. By combining these models one will get an AHU configured the way it should be.
In a next step we will think of a possibility to provide a configurable AHU model to the user via parameter tab.

Every component will be adressable in the way the old model adresses the different fuctions, by using set-Value conditions for the outlet temperature/humidity. Moreover a possibility to use heatPorts and mass flow rates will be provided to use the components as well for some control tests.

Closes #310 .

**Comment by PMehrfeld** [comment]
@KremerMartin : Since #310 is closed, does it mean this issue can also be closed?

**Comment by KremerMartin** [comment]
@PMehrfeld No. The intention was to close the old issue and provide more details here. The developed AHU model (until now developed in GitLab) will be transferred into a branch related to this issue on the next hackday.

**Comment by KremerMartin** [comment]
Added first version by commit 8bdd32b73519e23adfc9b0be678b662f32dbc14a

**Comment by XucYin** [comment]
@KremerMartin What is the current status? Maybe an issue for the hackday?

**Comment by KremerMartin** [comment]
@XucYin As I see it, there is no progress on this issue since the hackday at the beginning of this year. @KFinkbeiner Am I right with this assumption or have you provided some progress on the model by now?
Maybe I find some time during the hackday to make some progress.

**Comment by XucYin** [comment]
For the next steps I would suggest:
- [ ] check functionality of single components
- [ ] prepare every possible component combination and connections with condition expressions in one model
- [ ] define operation scenarios for comparison with the existing AHU model
- [ ] apply the modular AHU model in those scenarios, whose components and predefined internal connections can be activated/inactivated on demand using parameter selections
- [ ] search possible reasons for result deviations

**Comment by KremerMartin** [comment]
The comparison to the old `AixLib.Airflow.AirHandlingUnit.AHU` for a stationary cooling (only sensible) shows a difference of about 0.5 % (6 kW at a total of 1350 kW)
![grafik](https://user-images.githubusercontent.com/48821826/138270436-b87fce8f-fdc8-4576-b018-ad952ec8e8a8.png)

**Comment by KremerMartin** [comment]
Comparison for only humidification (adiabatic) shows good accordance:
![grafik](https://user-images.githubusercontent.com/48821826/141496922-e9806fd3-a8d5-4d19-9f58-0ca40240b2e6.png)
**Note: Only reproducable with the bug-fix for old ahu model (see #1193 )**

**Comment by FelixStege** [comment]
Any updates here? Maybe this is something for the upcoming hackday?

**Comment by KremerMartin** [comment]
Update: I cleaned the branch, deleted some old obsolete models and model parts.
Open ToDos:

- [ ] Check dehumidification case since difference in cooling demand and heating demand is quite high between models
- [ ] Add control for adiabatic humdification since Cooler is influenced by adiabatic humidification
- [ ] Check adiabatic humidification
- [ ] Check heating and cooling with heat recovery system

Since the old air handling unit model does not contain the functionalities for heat recovery only or steam humidification no verification will be possible for these cases.

**Comment by KremerMartin** [comment]
The current differences in the two ahu models:
![grafik](https://github.com/RWTH-EBC/AixLib/assets/48821826/43aad6b8-a0bb-4afb-964d-35ca83e52d1c)

**Comment by KremerMartin** [comment]
Dehumdification results get closer, if the bypass-factor dehumidification of old ahu is set to zero.
This is striking. Needs to be checked if old model is correctly implemented.

**Comment by KremerMartin** [comment]
Update on the progress:
Since results of the `AixLib.Airflow.AirHandlingUnit.AHU` are in many operation points not quite reliable, a comparison as planned seems not sensible.
Therefore the new models shall be compared to analogue models in the `Fluid` package.
First test cases for validation of the sub-components have been implemented, showing good agreement.
Open ToDos:

- [ ] Add validation case for adiabatic humidifier
- [ ] Add validation case for heat recovery system
- [ ] Add validtation case for fan
- [ ] Add control model for adiabatic humdification
- [ ] Set-up modular air handling unit model consisting of component and control models
- [ ] Set-up air handling unit model consisting of analogue component models in `Fluid` package
- [ ] Add validation case comparing both air handling unit models
- [ ] Check and refactor models with naming conventions
- [ ] Check and add documentations where missing

**Comment by KremerMartin** [comment]
closed with PR  #1437
